.github/actions/docker-build/action.yaml

```diff
@@ -36,6 +36,15 @@ inputs:
   push:
     description: Whether to push the image
     required: true
+  platform:
```

Comment:
Because the intent here is multiple arches should this be "platforms" ?

Reply:
This is intentional. This should be run for only run platform.
The idea is that this action gets triggered in workflow for each platform in parallel